src/shogun/mathematics/Math.h

```diff
@@ -1570,13 +1598,17 @@ class CMath : public CSGObject
 				SG_SERROR("CMath::parallel_qsort_index():: Not supported for complex128_t\n");
 			}
 
-
+		/// helper function for parallel_qsort_index.
 		template <class T1,class T2>
 			static void* parallel_qsort_index(void* p);
 
 
-		/* finds the smallest element in output and puts that element as the
-		   first element  */
+		/** finds the smallest element in output and puts that element as the
```

Comment:
Start with capitals (just to be consistent with the doc of the other functions).